README.md

```diff
@@ -87,10 +89,22 @@ Run tests:
 (venv) invoke test
 # or shorter
 (venv) inv test
+# or if using pipenv
+pipenv run inv test
 ```
 
 Check manifest:
 
 ```bash
 (venv) inv check-manifest
+# or if using pipenv
+pipenv run inv check-manifest
+```
+
+Clean out artefacts:
+
+```bash
+(venv) inv clean
+# or if using pipenv
+pipenv run inv clean
```

Comment:
How often, or when, should artifacts be cleaned?

Reply:
When building locally and mucking around with editable and non-editable installs, cleaning is a good habit. The artifacts cause a lot of weird behavior, that sapped good chunks of time for me this week. That being said, building is typically done via the GitHub Actions and local development (running tests) via directives mentioned doesn't typically need it. So I wouldn't worry about it too much on a day-to-day development. Only when doing more involved package debugging is this really useful.